test/ell1.d

```diff
@@ -11,3 +12,37 @@ unittest
             .shouldFailWith("left recursion");
     }
 }
+
+@("issue #11: disallow trailing content before EOF at LL(1) parser, by default")
+unittest
+{
+    with (sandbox)
+    {
+        const eag = `
+            NoEOF<+ 'Done': Done>: 'x'.
+            Done = 'Done'.
+            `.outdent;
+
+        run!"cat <<EOF | ./gamma --output-directory %s%sEOF"(directory, eag)
+            .shouldPassWith("NoEOF grammar is SLAG");
+        run!"cd %s && echo x yy zzz | ./NoEOF"(directory)
+            .shouldFailWith("syntax error, unexpected token\\(s\\) before end of file");
+    }
+}
+
+@("issue #11: allow trailing content before EOF at LL(1) parser if option -t has been passeed")
+unittest
+{
+    with (sandbox)
+    {
+        const eag = `
+            NoEOF<+ 'Done': Done>: 'x'.
```

Comment:
While the grammar is nicely reduced, a grammar for `a^n b^n`  would better motivate the issue: `a b b` is a valid input.

Reply:
Indeed, it's a more pathological case at all.

Reply:
The grammar example `a^n b^n` does not work for reproducing the issue behavior. To trigger the behavior, the last symbol in the grammar must be mandatory and not part of a repetition or option. 
At least, I could not reproduce this behavior with the mentioned grammar. So, I stay with the original minimal grammar.

Reply:
Got it.